src/main/java/nextstep/subway/line/domain/LineStation.java

```diff
@@ -99,4 +99,15 @@ public boolean isStartStation() {
     public boolean isPreStation(LineStation station) {
         return station.getStationId().equals(this.preStationId);
     }
+
+    public boolean isStationIdEquals(Long stationId) {
+        return this.stationId.equals(stationId);
+    }
+
+    public boolean isPreStation(Long stationId) {
+        if (preStationId == null) {
+            return stationId == null;
+        }
+        return this.preStationId.equals(stationId);
+    }
```

Comment:
LineStation의 preStationId 여부를 알기 위한 메서드 맞나요?
111 라인만으로 검증을 할 수 있을 것 같긴 한데...
혹시 null체크가 필요하다면 아래와 같은 구현은 어떨까요?
```suggestion
    public boolean isPreStation(Long stationId) {
        return this.preStationId == stationId;
    }
```

Reply:
둘다 primitive타입이 아닌 Long 객체 타입이라 == 비교를 안하고 다르게 처리했습니다. 
찾아보니 `Objects.equals(this.preStationId, stationId)`로 바꿀 수 있을 것 같아서 다음번에 적용해보겠습니다! ㅎㅎ